src-vue/src/plugins/store.ts

```diff
@@ -94,6 +95,11 @@ export const store = createStore<FlightCoreStore>({
             if (!state.developer_mode) {
                 store.commit('updateCurrentTab', Tabs.PLAY);
             }
+
+            let menu_bar_handle = document.querySelector('#fc_menu-bar');
+            if (menu_bar_handle !== null) {
+                menu_bar_handle.classList.toggle('developer_build');
+            }
```

Comment:
Toggling developer mode shouldn't change the top bar stripes IMO. The stripes indicate _debug build_ not that developer mode was activated.

The reason for this being that when I have local debug build running that I'm developing on rn as well as a release build, I want to be able to detect, which is which.

For example, one of these is a downloaded PR arctifact, the other one is locally compiled

![image](https://user-images.githubusercontent.com/40122905/228372909-6beb4cc3-2bef-47cc-956e-87bec3c2ee09.png)

do you know which one is which? ;)

Reply:
Oh I didn't know this style represented build state and not dev mode state.
Addressed in 9c5bfe23cc890ead0e94dab60edb70824fd80890